keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -177,12 +177,19 @@ rules.
 
 The policy instances will be ordered based on the numeric priority assigned to the
 CNP. A lower number corresponds to a higher order. For example, CNP with `priority` set
-to "0" will be considered as the highest priority policy.
+to "0" will be considered as the highest priority policy, while a large number, i.e.
```

Comment:
I would rather use Larger number == higher priority, but this is just a personal preference I'm ultimately fine with whatever the group agrees on

Reply:
Let's take this upstream to sig-net and ask for opinions there as well. There is already a TODO for this.